handbook/engineering/security/index.md

```diff
@@ -11,11 +11,14 @@ At Sourcegraph we think that security is an enabler for the business. Sourcegrap
 ## Responsibilities
 
 - Proactively improve the security of our application and infrastructure.
-- Implement automated security scanning of our codebase and infrastructrure, and resolve issues.
-    - https://github.com/sourcegraph/sourcegraph/security/code-scanning
+    - Identify what our security needs are and develop a roadmap to improve and ensure the security of our product.
```

Comment:
I liked this wording that you proposed in the JD, what do you think of using it here?

```suggestion
- Define, plan, and prioritize security work that needs to be done (and then go do that work).
```